canonical/canonicalwebteam.discourse
# Add flag to optionally limit redirects to url_prefix

## Done
By default, redirects are very restrictive, this change makes things more open by default, which is fine in 99% of our use-cases.

## QA
- You'll need to include this branch in your project.
- Add a couple of redirects to your discourse docs: 
  1. `/docs/test` -> `/docs`
  2. `/docs/test` -> `https://github.com`
- With your Docs initialised with `url_prefix="/docs"`:
  - `limit_redirects_to_url_prefix` set to `True` redirect 1. should work and redirect 2. should not.
  - `limit_redirects_to_url_prefix` unset (or set to `False`), both redirects should work

Fixes https://warthogs.atlassian.net/browse/WD-2871
